### ISSUE TITLE
Error after upgrading to version 1.5.1

### ISSUE THREAD

**Issue by krymen** [tlvince/eslint-plugin-jasmine#21]
After upgrading to v1.5.1 and running `lint` I get an error:
```
/home/krymen/workspace/my-project/ui/node_modules/eslint-plugin-jasmine/lib/prohibit.js:8
      var result = node.callee && node.callee.name.match(regex);
                                                   ^
TypeError: Cannot call method 'match' of undefined
```

**Comment by tlvince** [comment]
Hmm, I can't reproduce. Would you mind posting your `.eslintrc`, eslint version and your test file?

Alternatively, you could try modifying `/home/krymen/workspace/my-project/ui/node_modules/eslint-plugin-jasmine/lib/prohibit.js:8` with:

```diff
-var result = node.callee && node.callee.name.match(regex);
+var result = node.callee && node.callee.name && node.callee.name.match(regex);
```

**Comment by krymen** [comment]
It fails on every file I believe. Here is `.eslintrc`:
```{
  "env": {
    "jasmine": false
  },
  "rules": {
    "max-statements": 0,
    "no-global-strict": 0,
    "strict": [2, "never"],
    "one-var": false,
    "max-nested-callbacks": [4, 5],
    "new-cap": false,
    "func-style": false,
    "no-unused-vars": false,
    "angular/ng_definedundefined": false
  },
  "globals": {
    "browser": true,
    "should": true,
    "Q": true,
    "by": true,
    "element": true
  },
  "plugins": [
    "jasmine"
  ]
}```

Thankfully your fix works. I wonder why it is not detected by test?

**Comment by tlvince** [comment]
Not sure, but added that conditional. Will land in 1.5.2 shortly. Thanks!

**Comment by krymen** [comment]
Thanks! But still -- you missed a test proving that your fix works.

**Comment by tlvince** [comment]
See #22. :wink: Short on time right now.

**Comment by krymen** [comment]
Cool! ;)
